data/distro.yml

```diff
@@ -738,3 +738,25 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
+    
+- name: "PCLinuxOS"
+  logo: "pclinuxos.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>The flatpak package is available in the PCLinuxOS repositories, it can be installed using Synaptic Package Manager.</p>
```

Comment:
It really is not preinstalled? Anyway, if the next step is using a terminal, it would be better if the first step was also a terminal command.